zebra/zebra_vxlan_private.h

```diff
@@ -215,6 +215,15 @@ static inline void zl3vni_get_rmac(zebra_l3vni_t *zl3vni, struct ethaddr *rmac)
 		memcpy(rmac->octet, zl3vni->svi_if->hw_addr, ETH_ALEN);
 }
 
+struct host_rb_entry {
+	RB_ENTRY(host_rb_entry) hl_entry;
+
+	struct prefix p;
+};
+
+RB_HEAD(host_rb_entry_rb, host_rb_entry);
```

Comment:
I think the name `host_rb_entry_rb` is a bit confusing here. Maybe we could rename this to `host_tree` to convey its meaning more clearly.